.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -568,11 +568,14 @@
             Inputs="$(MSBuildThisFileFullPath);$(_ResizetizerTaskAssemblyName);$(_ResizetizerInputsFile);$(_MauiSplashInputsFile);@(AppxManifest)"
             Outputs="$(_MauiManifestStampFile);$(_MauiIntermediateManifest)Package.appxmanifest">
 
+        <_WindowsApplicationId Condition="'$(_WindowsApplicationId)' == '' and '$(ApplicationIdGuid)' != ''">$(ApplicationIdGuid)</_WindowsApplicationId>
+        <_WindowsApplicationId Condition="'$(_WindowsApplicationId)' == '' and '$(ApplicationId)' != ''">$(ApplicationId)</_WindowsApplicationId>
```

Comment:
Updated the logic to first try the ApplicationIdGuid but also fall back to the ApplicationId if none is found because of existing projects and also this property may be set via conditions. If it is a reverse domain, the task will fail with a relevant message.